spec/lib/convenient_service/rspec/matchers/custom/export_spec.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+require "convenient_service"
+# rubocop:disable RSpec/MultipleMemoizedHelpers
+RSpec.describe ConvenientService::RSpec::Matchers::Custom::Export do
+  subject(:matcher_result) { matcher.matches?(container) }
+
+  let(:matcher) { described_class.new(method_name, **kwargs) }
+
+  let(:method_name) { :bar }
+  let(:scope) { :class }
+  let(:kwargs) { default_kwargs }
+  let(:default_kwargs) { {scope: scope} }
+
+  let(:container) do
+    Class.new do
+      include ConvenientService::Support::DependencyContainer::Export
+
+      export :bar, scope: :class do
+        ":bar with scope: :class"
+      end
+
+      export :foo do
+        ":foo with scope: :instance"
+      end
+    end
+  end
+
+  describe "#matches?" do
+    context "when method is NOT exported" do
+      let(:method_name) { :non_existent }
+      let(:scope) { :class }
+
+      it "returns false" do
+        expect(matcher_result).to eq(false)
+      end
+    end
+
+    context "when scope is NOT passed" do
+      let(:method_name) { :foo }
+      let(:kwargs) { ConvenientService::Utils::Hash.except(default_kwargs, [:scope]) }
+
+      it "returns true" do
+        expect(matcher_result).to eq(true)
+      end
+    end
```

Comment:
**Intention**

<img width="720" alt="Screenshot 2023-02-07 at 13 47 46" src="https://user-images.githubusercontent.com/22632866/217236825-aa7fd65a-7abe-4189-8f84-c1b06f76bffc.png">

Reply:
Please, move inside `context "when method is exported" do`